satyr.spec.in

```diff
@@ -77,6 +77,8 @@ Python 3 bindings for %{name}.
 %setup -q
 
 %build
+autreconf
```

Comment:
```suggestion
autoreconf
```

Also, apparently, you need to add `satyr-version` to `EXTRA_DIST` in `Makefile.am`.